src/main/java/com/ericsson/ei/mongodbhandler/MongoDBHandler.java

```diff
@@ -127,6 +129,20 @@ public  boolean updateDocument(String dataBaseName, String collectionName, Strin
         return false;
     }
 
```

Comment:
Some description here too about what the method does and why we need it (because it finds and modifies record in one query).